vars/kolaTestIso.groovy

```diff
@@ -1,102 +1,9 @@
 // Run kola testiso
 // Available parameters:
-//     arch:               string  -- the target architecture
 //     cosaDir:            string  -- cosa working directory
-//     extraArgs:          string  -- extra arguments to pass to `kola testiso`
-//     extraArgs4k:        string  -- extra arguments to pass to 4k `kola testiso`
-//     extraArgsMultipath: string  -- extra arguments to pass to multipath `kola testiso`
-//     extraArgsUEFI:      string  -- extra arguments to pass to UEFI `kola testiso`
-//     scenarios:          string  -- scenarios to pass to `kola testiso`
-//     scenarios4k:        string  -- scenarios to pass to `kola testiso`
-//     scenariosMultipath: string  -- scenarios to pass to `kola testiso`
-//     scenariosUEFI:      string  -- scenarios to pass to `kola testiso`
-//     skipMetal4k:        boolean -- skip metal4k image
-//     skipMultipath:      boolean -- skip multipath tests
-//     skipUEFI:           boolean -- skip UEFI tests
-//     skipSecureBoot      boolean -- skip secureboot tests
-//     marker:             string  -- some identifying text to add to uploaded artifact filenames
```

Comment:
We still need these two knobs. `marker` is used by the `bump-lockfile` job which runs `kola testiso` for multiple arches in the same run (so needs a way to name the artifacts differently). `skipSecureBoot` is used by https://github.com/coreos/fedora-coreos-pipeline/blob/85a172f6f4f1774ac105ca1e429f3ec74b29a1e0/docs/config.yaml#L31. Ahh, unless you mean we'll use `--denylist-test` from the pipeline to do this now? Would probably be cleaner to encapsulate that here instead so it's consistent with `kola()` (which also takes a `skipSecureBoot` parameter).